crud/common/stash.lua

```diff
@@ -20,12 +20,20 @@ local stash = {}
 -- @tfield string stats_metrics_registry
 --  Stash for metrics rocks statistics registry.
 --
+-- @tfield string storages_info_on_select
+--  Stash for storages info during select working registry.
+--
+-- @tfield string select_module_compat_info
+--  Stash for select compatability version registry.
+--
 stash.name = {
     cfg = '__crud_cfg',
     stats_internal = '__crud_stats_internal',
     stats_local_registry = '__crud_stats_local_registry',
     stats_metrics_registry = '__crud_stats_metrics_registry',
     ddl_triggers = '__crud_ddl_spaces_triggers',
+    storages_info_on_select = '__storages_info_on_select',
```

Comment:
Why we adding new sections to stash?

Reply:
Deleted, done via `fiber storage` for merger and `iter.storages_info` for old_select